### ISSUE TITLE
ino-nav-drawer: missing aria-label

### ISSUE THREAD

**Issue by AnjaB93** [inovex/elements#388]
### Describe the bug

Icon buttons should have a  propper aria-label for screenreaders.

### To Reproduce

<img width="695" alt="Bildschirmfoto 2021-06-08 um 12 01 03" src="https://user-images.githubusercontent.com/51286604/121335665-17786c00-c91b-11eb-9c91-41cc5e027f2f.png">

**Comment by TobiasHeimGalindo** [comment]
closed by #1035
